app/controllers/sessions_controller.rb

```diff
@@ -1,19 +1,28 @@
 class SessionsController < ApplicationController
   def create
     user = User.find_by email: params.dig(:session, :email)&.downcase
```

Comment:
tách code này ra, handle user nil luôn đi, cho khỏe, chứ dùng tooasn tử an toàn làm j cho cực